tests/modules/gvcftools/extractvariants/test.yml

```diff
@@ -0,0 +1,7 @@
+- name: gvcftools extractvariants test_gvcftools_extractvariants
+  command: nextflow run ./tests/modules/gvcftools/extractvariants -entry test_gvcftools_extractvariants -c ./tests/config/nextflow.config  -c ./tests/modules/gvcftools/extractvariants/nextflow.config
+  tags:
+    - gvcftools/extractvariants
+    - gvcftools
+  files:
+    - path: output/gvcftools/test.vcf.gz
```

Comment:
Can't you check the md5sum for this file?

Reply:
It was a variable md5sum and I can't use the `contains` options since the output is gzipped